Index: обучение1
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>проверка\r\nеще проверка1238900
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/обучение1 b/обучение1
--- a/обучение1	(revision 7f2a9e25d4b552715ec2a5d13263fd1691b83bfb)
+++ b/обучение1	(date 1736873703739)
@@ -1,2 +1,2 @@
 проверка
-еще проверка1238900
\ No newline at end of file
+еще проверка1238900аа
\ No newline at end of file
Index: 123
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>p655666666ллл
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/123 b/123
--- a/123	(revision 7f2a9e25d4b552715ec2a5d13263fd1691b83bfb)
+++ b/123	(date 1736873703739)
@@ -1,1 +1,1 @@
-p655666666ллл
\ No newline at end of file
+p655666666лллаааа
\ No newline at end of file
